src/coreclr/System.Private.CoreLib/src/ILLink/ILLinkTrim.xml

```diff
@@ -5,10 +5,6 @@
       <method name="_RegisterFrozenSegment" />
       <method name="_UnregisterFrozenSegment" />
     </type>
-    <!-- Accessed via private reflection by an external tracing controller. -->
-    <type fullname="System.Diagnostics.Tracing.EventPipe*" />
-    <!-- The private Event methods are accessed by private reflection in the base EventSource class. -->
-    <type fullname="System.Diagnostics.Tracing.NativeRuntimeEventSource" />
```

Comment:
I think I understand why we can unroot the `EventPipe` types (see [discussion](https://github.com/dotnet/runtime/issues/43657#issuecomment-740243218).

But why can we unroot `NativeRuntimeEventSource`?

Reply:
Yes, EventPipe work has already being [submitted](https://github.com/dotnet/runtime/pull/48053).

My understanding is that we no longer require to root NativeRuntimeEventSource as well. @sywhang might be able to give more context on this

Reply:
@LakshanF For unrooting NativeRuntimeEventSource, can you either wait until my other PR related to NativeRuntimeEventSource is merged, or just remove the EventPipe type for now? We need to do some more testing around this. As you can see the Checked CI leg is failing because EventListeners are no longer getting the events.